apache/superset
# fix(explore): hide advanced analytics for non temporal xaxis

### SUMMARY
The superset feature allows users to choose a non-temporal column for the x-axis.
However, when a non-temporal column is selected, advanced analytics and forecast options are not compatible.

In order to address this, UI should dynamically hide these features when a non-temporal column is selected.
This commit introduces the visibility option in the ControlPanelSectionConfig, enabling the hiding of specific groups of controlSets.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

Before:

https://github.com/apache/superset/assets/1392866/19b5c98d-415d-4b4e-9454-c4eb21b5a65a

After:

https://github.com/apache/superset/assets/1392866/b1439a88-6090-482d-8c96-3de8610918ec

### TESTING INSTRUCTIONS
Choose non-temporal column in x-axis option
Verify that "Advanced analytics" and "Predictive Analytics" sections are hidden

### ADDITIONAL INFORMATION
